projects/sbb-esta/angular-business/src/lib/header/styles/_navButton.scss

```diff
@@ -0,0 +1,101 @@
+@mixin navButton() {
```

Comment:
Merge this into the navbutton.component.scss (Originally we had a different use case for these style files, but that is no longer necessary. The structure is merely legacy.)